qemu/tests/cfg/balloon_memhp.cfg

```diff
@@ -0,0 +1,41 @@
+# Notes:
+#   Need add movable_node in kernel line for rhel8 guest, rhel7 won't work well
+#   with memory unplug.
```

Comment:
Does that mean it requires manual configuration?

Reply:
Yes, unless we have a case to update kernel cmd line as we want.

Reply:
If so,  we need add one step in the script to check this argument in kernel line, and add it if it is missing.
You can refer module  `utils_test.check_kernel_cmdline` and `utils_test.update_boot_option`.

Reply:
The comment was removed and no update for guest kernel, could you explain more ?   No restrict with guest kernel now ?

Reply:
Because memory unplug part is removed from case based on dev's suggestion,  then we don't need the parameter now.